src/index.js

```diff
@@ -0,0 +1,93 @@
+import List from './list.js';
+import './styles.css';
+import removeTask from './removeTask.js';
+import handleCheckboxChange from './checkChange.js';
+
+const listTaskClass = new List();
+
+const taskInput = document.querySelector('#task-description');
+const taskListContainer = document.querySelector('#show-task');
+const addTaskForm = document.querySelector('#task-form');
+
+const createTaskEle = (task) => {
```

Comment:
- [ ] Kindly, it is recommended to make your function names completed and full-descriptive. It would be better to name your function `createTaskElement`.